### DISCUSSION TITLE
Full body avatars at readyplayer.me

### DISCUSSION THREAD

**Discussion by wftl** [mozilla/hubs#3203] [Avatar Questions] [answered]
Is there any way to use, and if not currently, is there any plan, or thought, to using full body avatars in Hubs, like the ones that can be generated over at https://fullbody.readyplayer.me/ ?

**Comment by robinkwilson** [comment, marked as answer]
Good question! We currently don't have plans for making full body avatars on our roadmap.

**Reply by normanwankowski** [reply]
Please put it on the road map. it is super important.

**Reply by angelcabrera-nr** [reply]
Please!

**Reply by mizne** [reply]
Please!

**Reply by marcio3dm** [reply]
Please!

**Reply by Fi1osof** [reply]
Please!
